README.md

```diff
@@ -14,6 +14,15 @@ It is recommended that you check out/use
 just using the main branch. We will regularly create new versions as bug fixes
 and features are added.
 
+## Command line tool usage
+
+```sh
+avifenc input.[jpg|png|y4m] -q 75 output.avif
```

Comment:
Nit: Move the `-q option` before the input file to match the short syntax help message:

```
$ ./avifenc
Syntax: avifenc [options] -q quality input.[jpg|jpeg|png|y4m] output.avif
where quality is between 0 (worst quality) and 100 (lossless).
Typical value is 60-80.

Try -h for an exhaustive list of options.
```

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2545